test/lib/analyzer/image-inspector.spec.ts

```diff
@@ -254,12 +265,6 @@ describe("getImageArchive", () => {
         path.join(imageSavePath, "image.tar"),
       );
       expect(imageExistsOnDiskAfterDelete).toBe(false);
-
-      const tmpFolderExistsOnDisk: boolean = fs.existsSync(imageSavePath);
-      expect(tmpFolderExistsOnDisk).toBe(false);
-
-      const customPathExistsOnDisk: boolean = fs.existsSync(customPath);
-      expect(customPathExistsOnDisk).toBe(true);
```

Comment:
Do we not need these assertions any more? Are we not interested in knowing if we leave any directories or similar behind us after `getImageArchive()` finishes?